android/quest/src/main/java/org/smartregister/fhircore/quest/ui/shared/components/SyncStatusView.kt

```diff
@@ -170,8 +170,9 @@ fun SyncBottomBar(
         contentAlignment = Alignment.Center,
       ) {
         val context = LocalContext.current
-        when (currentSyncJobStatus) {
-          is CurrentSyncJobStatus.Running -> {
+        when {
+          currentSyncJobStatus is CurrentSyncJobStatus.Running &&
+            appDrawerUIState.percentageProgress!! <= 100 -> {
```

Comment:
if the percent is >100 I think we still want to run the update, we want to change the percent to from >100 to 100